src/screens/AddNotificationEmailScreen/AddNotificationEmailScreen.tsx

```diff
@@ -0,0 +1,134 @@
+import { RouteProp } from '@react-navigation/native';
+import { StackNavigationProp } from '@react-navigation/stack';
+import React, { PureComponent } from 'react';
+import { Text, StyleSheet, View } from 'react-native';
+import { connect } from 'react-redux';
+
+import { images } from 'app/assets';
+import { Header, InputItem, ScreenTemplate, Button, FlatButton } from 'app/components';
+import { Route, PasswordNavigatorParams } from 'app/consts';
+import { createTc as createTcAction } from 'app/state/authentication/actions';
+import { createNotificationEmail as createNotificationEmailAction } from 'app/state/notifications/actions';
+import { typography, palette } from 'app/styles';
+
+const i18n = require('../../../loc');
+
+interface Props {
+  navigation: StackNavigationProp<PasswordNavigatorParams, Route.AddNotificationEmail>;
+  route: RouteProp<PasswordNavigatorParams, Route.AddNotificationEmail>;
+  createTc: () => void;
+  createNotificationEmail: Function;
+}
+
+type State = {
+  email: string;
+  error: string;
+};
+
+class AddNotificationEmailScreen extends PureComponent<Props, State> {
+  state = {
+    email: '',
+    error: '',
+  };
+
+  setEmail = (email: string): void => {
+    this.setState({
+      email,
+    });
+  };
+
+  onSave = () => {
+    //TODO: pass email and connect with api
+    this.props.navigation.navigate(Route.ConfirmNotificationCode);
+  };
+
+  skipAddEmail = () => {
+    const { createTc, createNotificationEmail, navigation } = this.props;
+    createTc();
```

Comment:
as far as I understand, `createTc()` redirects user to Terms & Conditions screen. If I'm right then the flow seems incorrect. It should be Terms & Conditions > Onboading > AddNotificationEmailScreen, right?

Reply:
createTc() is for add info this user finish and accepted TC

Reply:
Got it, nvm then.